cmd/lookout/event.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/src-d/lookout"
+	"github.com/src-d/lookout/service/bblfsh"
+	"github.com/src-d/lookout/service/git"
+	"google.golang.org/grpc"
+
+	gogit "gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+type EventCommand struct {
+	DataServer string `long:"data-server" default:"ipv4://localhost:10301" env:"LOOKOUT_DATA_SERVER" description:"gRPC URL to bind the data server to"`
+	Bblfshd    string `long:"bblfshd" default:"ipv4://localhost:9432" env:"LOOKOUT_BBLFSHD" description:"gRPC URL of the Bblfshd server"`
+	GitDir     string `long:"git-dir" default:"." env:"GIT_DIR" description:"path to the .git directory to analyze"`
```

Comment:
The description makes it sound like you need to provide `path/reponame/.git`, but it actually is `path/reponame`, right?

It might be more clear with "path to the Git repository to analyze"